src/org/jetuml/gui/EditorFrame.java

```diff
@@ -139,9 +135,12 @@ public EditorFrame(Stage pMainStage)
 		});
 
 		// Window position listener for notifications
-		ChangeListener<Number> stageMoveListener = (pObservableValue, pOldValue, pNewValue) -> updateNotificationPosition();
+		ChangeListener<Number> stageMoveListener = (pObservableValue, pOldValue, pNewValue) ->

```

Comment:
Should I keep this in the EditorFrame class? At first, it was in the main JetUML java file, I moved it when we moved the methods to the EditorFrame. Putting it in one file or the other has no functional impact, it's purely about style conventions.